sycl/test-e2e/Plugin/enqueue-arg-order-buffer.cpp

```diff
@@ -1,7 +1,7 @@
 // UNSUPPORTED: hip_nvidia
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple  %s -o %t.out
-// RUN: env SYCL_PI_TRACE=2 %GPU_RUN_PLACEHOLDER %t.out %GPU_CHECK_PLACEHOLDER
-// RUN: env SYCL_PI_TRACE=2 %CPU_RUN_PLACEHOLDER %t.out %CPU_CHECK_PLACEHOLDER
+// UNSUPPORTED: accelerator
```

Comment:
We support multiple `// UNSUPPORTED: ` directives?  ( As opposed to having to construct a boolean expression with `hip_nvidia` )

Reply:
Yes